solutionAD/tests/unit/SumCest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+include dirname(__DIR__, 2) . '/App/App.php';
+
+class sumCest
```

Comment:
Follow naming conventions: 
for start check https://www.php-fig.org/bylaws/psr-naming-conventions/